models/staging/src_iterable.yml

```
@@ -321,7 +321,7 @@ sources:
           - name: last_name
             description: Last name of the user
           - name: email_list_ids
-            description: Array of references to the lists associated with the user
+            description: "[DEPRECATED October 13, 2025 Use LIST_USER source instead.] Array of references to the lists associated with the user"
```

```suggestion
            description: [DEPRECATED October 13, 2025 Use LIST_USER source instead.] Array of references to the lists associated with the user
```
Just to keep it consistent